main.js

```diff
@@ -41,6 +41,8 @@ define(function (require, exports, module) {
 
     // Load initial document
     var defaultHTML = brackets.getModule("text!filesystem/impls/filer/lib/default.html");
+    var defaultCSS  = brackets.getModule("text!filesystem/impls/filer/lib/style.css");
+    var defaultJS   = brackets.getModule("text!filesystem/impls/filer/lib/script.js");
```

Comment:
I changed these to `require` instead of `brackets.getModule`. I think require is the more appropriate option. It's still showing 404 on the files though.